docs/install-manual.md

```diff
@@ -0,0 +1,144 @@
+
+# Create a swarm for development
+
+This section tells you how to create a Docker Swarm on your network to use only for debugging and testing purposes. You can also use this type of installation if you are developing custom applications for Docker Swarm or contributing to it.  
```

Comment:
Same comment here.

Unless you're toying with Swarm, you'll need to follow the manual process.